rust/private/rustc.bzl

```diff
@@ -518,6 +518,10 @@ def construct_arguments(
     add_native_link_flags(args, dep_info)
     add_crate_link_flags(args, dep_info)
 
+    if crate_info.type == "proc-macro" and not (crate_info.edition == None or crate_info.edition == "2015"):
```

Comment:
edition should never be None, if it is, it's a bug elsewhere

Reply:
Fabulous, changed to just a `!= "2015"` :) Thanks!